lib/twofishes/geocode_request.rb

```diff
@@ -0,0 +1,57 @@
+module Twofishes
+  class GeocodeRequest < ::GeocodeRequest
+    def initialize(options = {})
+      options[:ll] = prepare_ll(options[:ll])
+      options[:bounds] = prepare_bounds(options[:bounds])
+
+      options = substitute_aliases(options)
+
+      super(options)
+    end
+
+    def prepare_ll(ll)
+      case ll
+      when String
+        lat, lng = ll.split(/\s*,\s*/)
+        GeocodePoint.new(lat: lat.to_f, lng: lng.to_f)
+      when Array
+        GeocodePoint.new(lat: ll[0].to_f, lng: ll[1].to_f)
+      when Hash
+        GeocodePoint.new(lat: (ll[:lat] || ll['lat']).to_f, lng: (ll[:lng] || ll['lng'] || ll[:lon] || ll['lon']).to_f)
```

Comment:
Should we move handling the aliases to `substitute_aliases`? Also I would leave out the string keys. Same for bounds below.